packages/react/src/components/CalloutData/__stories__/CalloutData.stories.js

```diff
@@ -40,11 +40,24 @@ export default !DDS_CALLOUT_DATA
 export const Default = !DDS_CALLOUT_DATA
   ? undefined
   : ({ parameters }) => {
+      let data;
+      if (parameters.props.CalloutData.props.data.includes('%20')) {
+        data = parameters.props.CalloutData.props.data
+          .split('%20')
+          .map(elem => `${elem} `)
+          .join('');
```

Comment:
Do we need to care only about space character, or do we need full `decodeURIComponent()`?

Reply:
This problem is actually being caused but the percentage mark on the `data`, that if used with a space after it generates a double percentage mark on the url, that causes the bug.

I think that the main problem is the space character, but `decodeURIComponent` seems like a good solution as well